### PR TITLE
currency convertter

### PR DESCRIPTION
**Pull request by arash8877**

### CODE REVIEW

File: javascript/javascript3/week2/app.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  exchangeRates = data.rates;
+  const currenciesList = Object.keys(exchangeRates);
+  console.log(exchangeRates);
+  console.log(currenciesList);
+
+  const currencyFromDropdown = document.getElementById("currency-from");
+  const currencyToDropdown = document.getElementById("currency-to");
+
+  currenciesList.forEach((currency) => {
+  const option = document.createElement("option");
```
Comment (babak-f) [inline javascript/javascript3/week2/app.js:15]: For having a clear and readable code, it is a good idea to format your code before submitting. 
Here is how to do it in VS code: https://code.visualstudio.com/docs/editor/codebasics#_formatting

File: javascript/javascript3/week2/app.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+let exchangeRates;
+
+const getExchangeData = async () => {
+  const res = await fetch("https://open.er-api.com/v6/latest/USD");
+  const data = await res.json();
+  exchangeRates = data.rates;
+  const currenciesList = Object.keys(exchangeRates);
```
Comment (naresh699) [inline javascript/javascript3/week2/app.js:7]: please read about hoisting concept in js. also see either we should define variable in start of function or it work anywhere.

File: javascript/javascript3/week2/app.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  currenciesList.forEach((currency) => {
+  const option = document.createElement("option");
+  option.text = currency;
+  if (currency === "EUR") {
+    option.setAttribute("selected", true);
+  }
+  currencyFromDropdown.add(option);
+  });
+
+  currenciesList.forEach((currency) => {
```
Comment (babak-f) [inline javascript/javascript3/week2/app.js:23]: Try to combine the two forEach loops into one

File: javascript/javascript3/week2/index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
Comment (babak-f) [inline javascript/javascript3/week2/index.html:1]: Same applies here with formatting

File: javascript/javascript3/week2/style.css [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    transform: scale(0.97);
+    }
+
+    #output {
+        width: 10rem;
+        color: var(--clr--neutral-900);
+    }
+    
+    
+    @media (min-width: 600px) {
```
Comment (babak-f) [inline javascript/javascript3/week2/style.css:124]: Same with formatting

**Review by naresh699 — APPROVED** [review]
overall good work.

File: javascript/javascript3/week2/style.css [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+    body{
+    font-family: var(--ff-primary);
+    font-weight: 400;
+    font-size: 1.3rem;
+    line-height: 1.6;
+    }
+    
+    .container {
+    margin-inline: auto;
+    width: min(90%, 70.5rem);
```
Comment (babak-f) [inline javascript/javascript3/week2/style.css:26]: Use the min-width and max-width properties instead

File: javascript/javascript3/week2/style.css [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    margin: 1.8rem 0 4rem 0;
+    color: var(--clr--primary-200);
+    }
+    
+    .amount-wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-inline: auto;
+    width: max(40%);
```
Comment (babak-f) [inline javascript/javascript3/week2/style.css:53]: Use max-width instead. 
The max function will not work here anyways. Can you guess why? :)

File: javascript/javascript3/week2/style.css [hunk 8]
```diff
@@ -0,0 +1,155 @@
+:root {
+    --clr--primary-200: #F3EED9;
```
Comment (babak-f) [inline javascript/javascript3/week2/style.css:2]: Good to define variables 👍

File: javascript/javascript3/week2/style.css [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+    .bg-dark {
+    background-color: var(--clr--neutral-900);
+    color: var(--clr--neutral-100);
+    padding: 4rem 0;
+    display: flex;
+    height: 100vh;
+    }
+    
+    .bg-dark h2{
+    margin: 1.8rem 0 4rem 0;
```
Comment (babak-f) [inline javascript/javascript3/week2/style.css:44]: Using rem - nice 👍 
As a minor comment, I would suggest to only use multiples of 0.5 (or worst case 0.25), to keep the spacing consistent on the screen

File: javascript/javascript3/week2/app.js [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+  const option = document.createElement("option");
+  option.text = currency;
+  if (currency === "DKK") {
+    option.setAttribute("selected", true);
+  }
+  currencyToDropdown.add(option);
+  });
+  
+  };
+  getExchangeData();
```
Comment (babak-f) [inline javascript/javascript3/week2/app.js:33]: For functions like this it is better to add them to the window.onload listener. That way you can be sure they will be called at the right time, i.e. once the page is fully loaded. 

See https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event

**Comment by arash8877** [comment]
Hi, Thank you so much for your feedback.

kind regards
Arash

**Pull request merged**